dm/worker/subtask.go

```diff
@@ -110,11 +110,6 @@ func NewSubTaskWithStage(cfg *config.SubTaskConfig, stage pb.Stage, etcdClient *
 
 // Init initializes the sub task processing units
 func (st *SubTask) Init() error {
-	cfgDecrypted, err := st.cfg.DecryptPassword()
```

Comment:
when `Resume` will call `Init` again, so will DecryptPassword twice

Reply:
https://github.com/pingcap/dm/blob/187935150417195fa738bf6a73a0107fa0b91fa1/dm/worker/subtask.go#L423 
https://github.com/pingcap/dm/blob/187935150417195fa738bf6a73a0107fa0b91fa1/dm/worker/subtask.go#L183
`Resume` -> `Run` -> `Init`

Reply:
is `if !st.initialized.Get()` will be `false` when resuming? https://github.com/pingcap/dm/blob/187935150417195fa738bf6a73a0107fa0b91fa1/dm/worker/subtask.go#L422

Reply:
but may set false here when initial failed https://github.com/pingcap/dm/blob/e31e5de0c0ad0c5a399b41afb09e6d4768f37802/dm/worker/subtask.go#L121